### ISSUE TITLE
Background

### ISSUE THREAD

**Issue by arrellc** [ivan-usov/photdiag#5]
Often a FEL shot is dropped in a repeating or random way to take FEL off background shots.  When this occurs this will be written into the event codes.

An option to read from h5 or json file while a user specified event code which is saved in the bs data could allow these FEL off shots to be used as a background.

**Comment by arrellc** [comment]
To be more specific:
in the .bsread.h5 file for a step is a channel with event codes (SAR-CVME-TIFALL5:EvtSet), under data  is a 2d array where the column index is the event code and the row is a pulse.

Event code (column) 12 is at the FEL rate, and event code 25 is what is called a laser dark shot where the laser timing is delayed to the FEL - this gives us a background image on the time tool.

Simply put when there is a 1 in column 25 this is a background shot.

User specified information could be event code channel name (will likely stay the same once specified per end station), FEL event code, and laser delay event code (background).

![image](https://user-images.githubusercontent.com/33692029/53629939-1d952380-3c0f-11e9-8992-805df9be1437.png)
